packages/ketcher-core/src/domain/serializers/mol/v2000.js

```diff
@@ -256,6 +256,12 @@ function parsePropertyLines(ctab, ctabLines, shift, end, sGroups, rLogic) {
           const sGroupId = Number(eg) - 1
           sGroups[sGroupId].data.expanded = true
         })
+      } else if (type === 'SAP') {
```

Comment:
Can we add a `enum` for this with comments? I believe, that it is not clear for a person, who sees this for the first time.

Reply:
could you create issue for refactoring? please for this. It will affect lot of files and worth to do out of this PR.

Reply:
I will create issue

Reply:
added: https://github.com/epam/ketcher/issues/2797